modules/chat/client/controllers/chat.client.controller.js

```diff
@@ -10,6 +10,8 @@ angular.module('chat').controller('ChatController', ['$scope', '$location', 'Aut
     $scope.classes = [];
     $scope.currentlySelectedClassID;
     $scope.isDefaultSet = false;
+    $scope.currentlySelectedCourseCode;
+
```

Comment:
Could you provide the comment for what $scope.currentlySelectedCourseCode is for ? We also have  $scope.currentlySelectedClassID which is used to switch the chat room.

Reply:
I only created the currentlySelectedCourseCode. The currentlySelectedClassID was already there and it's the generated id from the database to determine the currently selected class for the chat. But I needed to display the actual course code so I created the other variable to display the current course code.